From ae241feec49dd9b9fc1355e4be37c3f8b268de87 Mon Sep 17 00:00:00 2001
From: Peter Lemenkov <lemenkov@gmail.com>
Date: Thu, 18 Mar 2010 14:57:21 +0300
Subject: [PATCH 06/10] Change directory to readable by everyone

This requires to suppress error messages like
"File operation error: eacces". See rhbz #564686:

https://bugzilla.redhat.com/564686

Signed-off-by: Peter Lemenkov <lemenkov@gmail.com>
---
 src/ejabberdctl.template |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/src/ejabberdctl.template b/src/ejabberdctl.template
index bceec28..ee74650 100644
--- a/src/ejabberdctl.template
+++ b/src/ejabberdctl.template
@@ -355,6 +355,10 @@ wait_for_status()
     return $status
 }
 
+# In order to prevent issues with "File operation error: eacces."
+# See rhbz #564686.
+cd /
+
 case $ARGS in
     ' start') start;;
     ' debug') debug;;
-- 
1.7.2.1

